orion-viewer/version.properties

```diff
@@ -1,2 +1,2 @@
-orion.version.code=198
-orion.version.name=0.81.3
\ No newline at end of file
+orion.version.code=199
```

Comment:
I do +4 increment, as there are 4 release builds that are published on Google Play
I will fix it by my own

Reply:
Fixed